osbrain/tests/test_common.py

```diff
@@ -30,3 +31,36 @@ def test_loglevel():
     # Invalid initialization
     with pytest.raises(ValueError):
         LogLevel('FOO')
+
+
+def test_topics_to_bytes():
+    """
+    Test topics_to_bytes basic usage and required types.
+    """
+    handlers = {'topic_one': 'handler', b'topic_two': 'handler'}
+
+    # Valid call (without uuid)
+    curated = topics_to_bytes(handlers)
+    assert isinstance(curated, dict)
+
+    for k, v in curated.items():
+        assert isinstance(k, bytes)
+        assert k.startswith(b'topic_')
+        assert v == 'handler'
+
+    # Valid call (with uuid)
+    curated = topics_to_bytes(handlers, uuid=b'U')
+    assert isinstance(curated, dict)
+
+    for k, v in curated.items():
+        assert isinstance(k, bytes)
+        assert k.startswith(b'Utopic_')
+        assert v == 'handler'
```

Comment:
Maybe it could be simplified with some:

```python
@pytest.mark.parametrize('uuid', [None, b'U'])
```

To avoid repeating almost the same code.

Otherwise, I would rather split this test into two: one for `str`, another for `bytes`. But I think a parametrization makes more sense in this case.